packages/core/src/orders/redux/selectors.js

```diff
@@ -573,3 +573,43 @@ export const isOrderItemAvailableActivitiesLoading = state =>
  */
 export const getOrderItemAvailableActivitiesError = state =>
   getOrderItemAvailableActivities(state.orders).error;
+
+/**
+ * Returns a list of order summaries, with the order items if the
+ * details of the order were previously fetched.
+ *
+ * @function
+ * @param {string} orderId - Order id.
+ * @param {object} state - Application state.
+ * @returns {object} Order summaries array.
+ */
+export const getOrderShipments = (state, orderId) => {
+  const order = getOrder(state, orderId);
+
+  if (!order) return;
+
+  // Omit both byMerchant and items, we don't want to expose all of the items
+  // of the order by its alphanumeric, we split the order items by the order summary
+  const { byMerchant, items, ...sharedOrderInformation } = order;
+  const result = [];
+
+  for (const merchantId in byMerchant) {
+    for (const merchantOrderCode in byMerchant[merchantId]) {
```

Comment:
I think this name is misleading, as this variable will contain the keys inside the `byMerchant[merchantId]` object which is different from the `merchantOrderCode` property which exists inside the order object. What do you think of calling this just `key` or another more general name?

Reply:
The name is correct, this selector only works for the order split flow. So when the `splitByMerchantOrderCode` inside of the `byMerchant[merchantId]` object there will be other objects using the merchantOrderCode as a key.

I’ll add an extra comment to warn the devs that this should only be used when the `doGetOrders` is provided the prop `splitByMerchantOrderCode` as true. You can find more about this inside of that action.

Reply:
Added an extra comment in the selector comment, please check if it’s clearer.

Reply:
I see, I was working on the `next` branch which still does not have that split logic. Thanks!